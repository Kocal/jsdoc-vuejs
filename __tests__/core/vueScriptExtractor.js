const path = require('path')
const extractVueScript = require('../../lib/core/vueScriptExtractor')

describe('core.extractVueScript', () => {
  const filename = path.join(__dirname, '__fixtures__', 'Component.vue')

  test('extract script', () => {
    expect(extractVueScript(filename)).toMatchSnapshot()
  })
})
