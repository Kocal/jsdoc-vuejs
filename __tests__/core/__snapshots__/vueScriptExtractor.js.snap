// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core.extractVueScript extract script 1`] = `
"
/**
 * @vue-data {String} [foo=bar] Foo description
 */
export default {
  name: 'Component',
  data() {
    return {
      foo: 'bar'
    }
  }
}
"
`;
